virt/virt-4-12-release-notes.adoc

```diff
@@ -212,6 +212,24 @@ link:https://access.redhat.com/support/offerings/techpreview[Technology Preview
 
 [id="virt-4-12-known-issues"]
 == Known issues
+
+// Fix targeted for 4.12.1
+* The `TopoLVM` provisioner name string has changed in {VirtProductName} {VirtVersion}. As a result, the automatic import of operating system images might fail with the following error message (link:https://bugzilla.redhat.com/show_bug.cgi?id=2158521[*BZ#2158521*]): 
```

Comment:
If this does get fixed in 4.12.1, this won't be an issue, but I might s/{VirtVersion}/4.12 just in case.